ext/openssl/openssl.c

```diff
@@ -967,6 +967,21 @@ static void php_openssl_dispose_config(struct php_x509_request * req TSRMLS_DC)
 }
 /* }}} */
 
+#ifdef PHP_WIN32
+#define PHP_OPENSSL_RAND_ADD_TIME() ((void) 0)
+#else
+#define PHP_OPENSSL_RAND_ADD_TIME() php_openssl_rand_add_timeval()
+#endif
+
+static inline void php_openssl_rand_add_timeval()  /* {{{ */
```

Comment:
Since this is not used on Windows, it might as well be apart of the #else part ^